src/v2/Apps/Artist/artistRoutes.tsx

```diff
@@ -144,10 +147,37 @@ export const artistRoutes: AppRouteConfig[] = [
         prepare: () => {
           AuctionResultsRoute.preload()
         },
+        prepareVariables: ({ artistID }, props) => {
+          const urlFilterState = paramsToCamelCase(
+            props.location ? props.location.query : {}
+          )
+
+          return {
+            artistID,
+            ...initialAuctionResultsFilterState(0, 0),
```

Comment:
Understand that the `initialAuctionResultsFilterState` was created before this PR, but we might make this more readable by converting the argument into an object. The `(0, 0)`, combined with the function name, doesn't convey any information, where 
```ts
initialAuctionResultsFilterState{{ startDate: 0, endDate: 0 })
```
(or something) immediately makes sense.

Reply:
`startDate` and `endDate` makes it way easier to read 👍